end_to_end/tpu/llama2/7b/test_llama2_7b.sh

```diff
@@ -71,3 +71,6 @@ export NEW_CKPT_PATH=${BASE_OUTPUT_DIRECTORY}/${PARAMETER_CHECKPOINT_RUN}/checkp
 
 # We run decoding on the fine-tuned parameter checkpoint
 python3 MaxText/decode.py MaxText/configs/base.yml load_parameters_path=${NEW_CKPT_PATH} run_name=runner_decode_finetuned_${idx} base_output_directory=${BASE_OUTPUT_DIRECTORY} per_device_batch_size=1 model_name='llama2-7b' ici_autoregressive_parallelism=4 max_prefill_predict_length=4  max_target_length=16 prompt="I love to" attention=dot_product scan_layers=false
+
+# We also test whether the forward pass logits match the golden logits for Llama2-7b
+python3 MaxText/tests/fwd_pass_testutil.py  MaxText/configs/base.yml load_parameters_path=${UNSCANNED_CKPT_PATH} run_name=forward_pass_test per_device_batch_size=1 model_name=llama2-7b ici_tensor_parallelism=4 max_prefill_predict_length=4 max_target_length=4 dataset_type=synthetic  dtype=float32 scan_layers=false
```

Comment:
probably we should pass the prompt as an argument? just consider if the default prompt changes.

Reply:
The prompts and the golden logits are all rolled in the  input `jsonl` file under `test_assets`. 

Currently the 3 prompts for this logit checker script are `I love to`, `Today is a`, `What is the`

We are not computing the golden logits live during the test, we are keep the prompts+golden logits saved and only running MaxText logits during the test.